### PR TITLE
[#73] getting fee per kb from chainz.cryptoid.info

### PR DESCRIPTION
**Pull request by trojkat**

### CODE REVIEW

File: clove/network/base.py [hunk 1]
```diff
@@ -228,19 +228,40 @@ def get_new_wallet(cls):
...
+            if not tx_fee:
+                continue
+
+            tx_fee_per_kb = (tx_fee * 1000) / tx_size
+            fees.append(tx_fee_per_kb)
+
+        if not fees:
             return
 
+        return round(sum(fees) / len(fees), 8)
```
Comment (szerepak) [inline clove/network/base.py]: maybe it's worth to import `numpy.mean` ?

File: clove/network/bitcoin.py [hunk 2]
```diff
@@ -181,7 +180,7 @@ def publish(self):
     @property
     def size(self) -> int:
         """Returns the size of a transaction represented in byte."""
```
Comment (szerepak) [inline clove/network/bitcoin.py]: docstring is outdated

File: clove/utils/external_source.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        logger.exception(e)
+        return
+
+    if resp.status != 200:
+        logger.debug(
+            'Could not get last transactions for %s network, status code: %s',
+            network, resp.status)
+        return
+
+    last_transactions = [t['hash'] for t in json.loads(resp.read().decode())]
```
Comment (szerepak) [inline clove/utils/external_source.py]: `return` here

File: clove/utils/logging.py [hunk 4]
```diff
@@ -1,3 +1,12 @@
+import logging
+
+import coloredlogs
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger()
+coloredlogs.install(logger=logger, level=logging.DEBUG)
```
Comment (szerepak) [inline clove/utils/logging.py]:
```
from clove.constants import COLORED_LOGS_STYLES
coloredlogs.install(logger=logger, level=logging.DEBUG, level_styles=COLORED_LOGS_STYLES)
```

**Review by szerepak — APPROVED** [review]

File: clove/network/base.py [hunk 5]
```diff
@@ -228,19 +228,40 @@ def get_new_wallet(cls):
...
+                continue
+
+            tx_fee = get_transaction_fee(network, tx_hash)
+            if not tx_fee:
+                continue
+
+            tx_fee_per_kb = (tx_fee * 1000) / tx_size
+            fees.append(tx_fee_per_kb)
+
+        if not fees:
```
Comment (szerepak) [inline clove/network/base.py]: also `return numpy.mean(fees) if fees else None`

**Review by pafff — APPROVED** [review]

**Pull request merged**